src/DssSpell.t.base.sol

```diff
@@ -2406,7 +2406,7 @@ contract DssSpellTestBase is DSTest, DSMath {
         psm.sellGem(address(this), amount);
         amount -= amount * tin / WAD;
         assertEq(token.balanceOf(address(this)), 0);
-        assertEq(dai.balanceOf(address(this)), amount);
+        assertEq(dai.balanceOf(address(this)), amount * (10 ** (18 - token.decimals())));
```

Comment:
you could follow a similar pattern introduced in #203 `checkIlkIntegration` to generalize `checkPsmIlkIntegration` as well by replacing `amount` with `amount18`:
`uint256 amount18 = token.decimals() == 18 ? amount : amount * 10**(18 - token.decimals());`

Reply:
I think this is fine though as `10 ^ (18 - 18) = 10 ^ 0 = 1`, so it's a no-op.

Reply:
yeah, just a bit more elegant aesthetic I think and to keep the naming consistent for the token-to-dai test block, and improve readability.

Reply:
also, I think if you keep the current naming convention, in the dai-to-token block you'll use a reduced about as `tin` is subtracted in the token-to-dai block.

Reply:
Yeah no need of the conditional statement.

Reply:
yeah, It can be removed in `checkIlkIntegration`, but I would add the `amount18` variable in the `checkPsmIlkIntegration` just for readability.